### PR TITLE
Allow http-api-data 0.4.1

### PR DESCRIPTION
**Pull request by felixonmars**
Builds fine and all tests pass here.

### CODE REVIEW

File: servant/servant.cabal [hunk 1]
```diff
@@ -98,7 +98,7 @@ library
   -- We depend (heavily) on the API of these packages:
   -- i.e. re-export, or allow using without direct dependency
   build-depends:
-      http-api-data          >= 0.4      && < 0.4.1
+      http-api-data          >= 0.4      && < 0.4.2
```
Comment (phadej) [inline servant/servant.cabal]: We depend on tight range, as the API is re-exported.

Please bump lower bounds to `>=0.4.1`

Comment (felixonmars) [reply servant/servant.cabal]: Updated!

**Comment by phadej** [comment]
I fixed the `master` CI, so if you mind rebasing to turn the red mark into green one.

Sorry for inconvenience, I didn't notice the failure at first.

**Comment by felixonmars** [comment]
Sure, rebased :)

**Pull request merged**